pyproject.toml

```diff
@@ -43,36 +43,36 @@ ansys-openapi-common = "< 2.0.0"
 ansys-grantami-bomanalytics-openapi = "1.1.0"
 
 # Packages for the examples extra
-jupyterlab = { version = "3.6.5", optional = true }
-lxml = { version = "4.9.3", optional = true }
-pandas = { version = "1.3.5", optional = true }
-tabulate = { version = "0.9.0", optional = true }
+jupyterlab = { version = "^3.6.5", optional = true }
+lxml = { version = "^4.9.3", optional = true }
+pandas = { version = "^1.5.0", optional = true }
```

Comment:
Can we use pandas 2.0.0? Or is this a Python 3.8 thing?

Reply:
I think we can, let me see